docs/index.rst

```diff
@@ -17,6 +17,16 @@ that redirects to a static file service.
 Advanced users can configure a search appliance to support "docker search" and
 can setup repository protection using SSL certificates.
 
+Docker pull
+-----------
+
+There is a possibility to ``docker pull`` the image from Crane by its tag or manifest digest.
```

Comment:
It is possible to "docker pull" an image from Crane by its tag or manifest digest.